builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -439,48 +447,48 @@ function store.get_formspec(dlgdata)
 
 		-- buttons
 		local description_width = W - 0.375*5 - 1 - 2*1.5
```

Comment:
This can be moved down to immediately where it is moved.

Also, there is unused space. The red line (on the right) should be the same as the green line (on the left).

![image](https://user-images.githubusercontent.com/2122943/103447592-2c264900-4c85-11eb-838f-d645e350b27a.png)

(left/right included for colorblind readers)